src/OrchardCore.Modules/OrchardCore.OpenId/Services/OpenIdAuthorizationStore.cs

```diff
@@ -348,10 +349,23 @@ public virtual Task<OpenIdAuthorization> InstantiateAsync(CancellationToken canc
         /// A <see cref="Task"/> that can be used to monitor the asynchronous operation,
         /// whose result returns all the elements returned when executing the specified query.
         /// </returns>
-        public virtual Task<ImmutableArray<OpenIdAuthorization>> ListInvalidAsync(int? count, int? offset, CancellationToken cancellationToken)
+        public virtual async Task<ImmutableArray<OpenIdAuthorization>> ListInvalidAsync(int? count, int? offset, CancellationToken cancellationToken)
         {
-            // TODO Sébastien Ros.
-            throw new NotImplementedException();
+            IQuery< OpenIdAuthorization> query = _session.Query<OpenIdAuthorization, OpenIdAuthorizationIndex>(authorization => authorization.Status != OpenIddictConstants.Statuses.Valid ||
```

Comment:
@sebastienros I'm glad you implemented that yourself :sweat_smile: 

Merci.

Reply:
I haven't tried it. I am also changing the identifiers to logical ones right now.